src/ws_transport.ts

```diff
@@ -30,9 +30,9 @@ import {
   INFO,
   NatsError,
   render,
-} from "https://raw.githubusercontent.com/nats-io/nats.deno/v1.14.0/nats-base-client/internal_mod.ts";
+} from "https://raw.githubusercontent.com/nats-io/nats.deno/js-module/nats-base-client/internal_mod.ts";
 
-const VERSION = "1.15.0";
+const VERSION = "1.16.0-1";
```

Comment:
the 0-1 looks little weird. A typo, or done on purpose?

Reply:
that is a qualifier - we can release 1.16.0 without conflict.